pkg/synopsysctl/cmd_serve_UI.go

```diff
@@ -1,828 +0,0 @@
-/*
```

Comment:
- We should also delete pkg/synopsysctl/a_synopsysctl-packr.go
- We should also edit the Makefile to not build the ui anymore: https://github.com/blackducksoftware/synopsys-operator/blob/master/Makefile#L99

Reply:
done @mphammer